tests/benchmark/run_benchmark.js

```diff
@@ -110,16 +123,22 @@ function run_multiple_times(cmd) {
     try {
         for (let i = 0; i < multirun; i++) {
             let start = performance.now();
-            execSync(cmd);
+            let ret = execSync(cmd);
             let end = performance.now();
             elapsed = (end - start);
             elapse_arr.push(elapsed);
+            ret = ret.toString().trim();
```

Comment:
Had better also check the return code of the process

Reply:
Resolved